### PR TITLE
Update libsql libraries

### PR DESCRIPTION
**Pull request by haaawk**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "@craftzdog/react-native-buffer": "^6.0.5",
+    "bob": "^2.7.0",
```
Comment (ospfranco) [inline example/package.json:15]: This looks incorrect

File: ios/libsql.xcframework/Info.plist [hunk 2]
```diff
@@ -10,34 +10,34 @@
 			<key>HeadersPath</key>
 			<string>Headers</string>
 			<key>LibraryIdentifier</key>
-			<string>ios-arm64_x86_64-simulator</string>
+			<string>ios-arm64</string>
```
Comment (ospfranco) [inline ios/libsql.xcframework/Info.plist:13]: This seems to have lost the x86_64 version of the library?

Comment (ospfranco) [reply ios/libsql.xcframework/Info.plist:13]: It's probably fine though

**Review by ospfranco — CHANGES_REQUESTED** [review]

**Comment by ospfranco** [comment]
The checks are failing, can you revert the change to package.json where you set libsql: true

File: example/package.json (hunk 1 above)
Comment (haaawk) [reply example/package.json]: This was accidentally commited - sorry. I removed it.

**Review by ospfranco — APPROVED** [review]

**Pull request merged**